rust/Cargo.toml

```diff
@@ -40,8 +40,8 @@ verbose_file_reads = "warn"
 # This profile optimizes for runtime performance and small binary size at the expense of longer build times.
 # Compared to default release profile, this profile reduced binary size from 29MB to 21MB
 # and increased build time (with only one line change in code) from 12 seconds to 133 seconds (tested on Mac M2 Max).
-[profile.release]
-lto = "fat"
+# [profile.release]
```

Comment:
no longer valid?

Reply:
Reverted. Had commented it out for faster image builds on laptop.